reddit.js

```diff
@@ -1,5 +1,52 @@
 var reddit = require("./data.json")
 
-// Write your solutions below
+var titleSub = reddit.data.children
+
+//1. logs all titles
+
+var allTitles = [];
+
+var title = titleSub.forEach(function(array) {
```

Comment:
Here, since titleSub is our array, we are actually iterating over each child object within that array. Because of this, ```child``` or ```post``` might be a more accurate parameter name here.